sections/plate_notation.tex

```diff
@@ -73,34 +73,38 @@ \subsection{Author-Topic \gls{lda} and Category-Topic \gls{lda}}
 \end{figure*}
 
 \subsection{Pachinko Allocation}
-In order to handle the hierarchical structure of the taxonomy metadata field, we will be using a hierarchical topic model, namely pachinko \citet{li2006pachinko}.
-Pachinko allocation generalizes \gls{lda}, to be able to operate on any DAG structure.
+In order to handle the hierarchical structure of the taxonomy metadata field, we will be using a hierarchical topic model, namely the \acrlong{pam} from \citet{li2006pachinko}.
+Pachinko allocation generalizes \gls{lda}, to be able to operate on any \gls{dag} structure.
 
-The DAG structure represents topics which in pachinko are distributions over words and/or other topics.
-It is set up in a tree structure with the first layer being the root node, a topic which all documents are part of, and the bottom layer consisting of leaf nodes, which are words. 
+The \gls{dag} structure represents topics which in pachinko allocation are distributions over words and/or other topics further down in the \gls{dag} structure, rather than distribution only over words as with \gls{lda}.
```

Comment:
```suggestion
The \gls{dag} structure represents topics which in pachinko allocation are distributions over words and/or other topics further down in the \gls{dag} structure, rather than a distribution only over words as with \gls{lda}.
```